nodes/src/nodes/tool_vectordb/IInstance.py

```
@@ -0,0 +1,321 @@
+            raise ValueError('delete: no valid object IDs provided')
+
+        store.remove(clean_ids)
+        return {'success': True, 'deleted_count': len(clean_ids)}
+
+
+def _normalize_input(input_obj: Any) -> Dict[str, Any]:
+    """Normalize tool input into a plain dict."""
+    if input_obj is None:
+        return {}
+
+    if hasattr(input_obj, 'model_dump') and callable(getattr(input_obj, 'model_dump')):
```

Ruff (0.15.9)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)